src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewerUtils.java

```diff
@@ -48,12 +48,11 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
-import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
 public class ProfileViewerUtils {
-	static Map<String, ItemStack> playerSkullCache = new HashMap<>();
+	static final HashMap<String, ItemStack> playerSkullCache = new HashMap<>();
```

Comment:
When the concrete implementation is not relevant, one should prefer the interface over the specific type.